docs/en/observability/synthetics-visualize.asciidoc

```diff
@@ -5,38 +5,61 @@
 <titleabbrev>Visualize</titleabbrev>
 ++++
 
-[WARNING]
-====
-This functionality is experimental and may be changed or removed completely in a future release.
-Elastic will take a best effort approach to fix any issues, but experimental features are not subject
-to the support SLA of official GA features.
-
-Have a question? Want to leave feedback? Visit the
-https://discuss.elastic.co/tags/c/observability/uptime/75/synthetics[Synthetics discussion forum].
-====
-
-Synthetic monitoring journeys can be visualized in the {uptime-app} side-by-side with
+beta[] Synthetic monitoring journeys can be visualized in the {uptime-app} side-by-side with
 your other Uptime monitors.
 
 [role="screenshot"]
 image::images/synthetic-app-overview.png[Synthetics app overview]
 
-Select a specific journey to expand it.
-The following details are provided for each step:
+[[analyze-synthetic-journeys]]
+== Analyze synthetic journeys
+
+On the *Monitors* table, you can click a listed journey to analyze it further
+on the monitor *Overview* page. You can examine journey details, including its
+availability, monitor ID, service availability per monitoring location, the timing
+for each synthetic check performed, and check statuses over time.
+
+The *History* table lists the total count of all of the synthetic checks that completed or failed
+during the journey. The timestamp for each check is also displayed, along with its duration.
+For any failed checks, the error and the failed step are displayed.
+
+[role="screenshot"]
+image::images/synthetics_overview.png[Synthetics overview page]
+
+[[review-synthetic-checks]]
+== Review synthetic checks
+
+To drill down and review the steps performed by a synthetic check, select it in the *History*
+table to view the synthetic check details page.
 
-* Step name
-* Step script
-* A screenshot taken during the step
+For each synthetic step, the following details are displayed:
+
+* The status of each step: `succeeded`, `failed`, or `skipped`.
+* Step name.
+* A screenshot taken of the step.
```

Comment:
You could also call out that we include the last good screenshot for a failed step (to make it easier to compare the failed step with the prior time it was good).

Reply:
Thanks Paul. I made that update.